src/controllers/IngredientController.py

```diff
@@ -0,0 +1,62 @@
+from flask import (Blueprint, jsonify, request)
+from src.models.Ingredient import Ingredient
+
+from src.services.IngredientService import IngredientService
+
+
+bp = Blueprint("ingredients", __name__, url_prefix="/ingredients")
+service = IngredientService()
+pass
+
+
+@bp.route("/", methods=["GET"])
+def get_all_ingredients():
+    data = service.getAllIngredients()
+    return jsonify({
+        "status": "Success",
+        "data": {
+            "ingredients": data
+        }
+    }), 200
+
+
+@bp.route("/ingredient", methods=["POST"])
```

Comment:
is **/ingredient** at the end of the path necessary/relevant? for posting a new ingredient we could use the same route "/" as for get all